.pre-commit-config.yaml.jinja

```
@@ -122,6 +122,13 @@ repos:
     rev: {{ repo_rev.setuptools_odoo }}
     hooks:
       - id: setuptools-odoo-make-default
+      - id: setuptools-odoo-get-requirements
+        args: [
+          "--output",
+          "requirements.txt",
+          "--header",
+          "# generated from manifests external_dependencies"
+        ]
```

Please don't:

```suggestion
        args:
          - --output
          - requirements.txt
          - --header
          - "# generated from manifests external_dependencies"
```

We don't use JSON for a reason :grin:

---

If you prefer. But that's nitpicking. It's valid yaml :)